frontend/apps/marketing/src/components/contentful/activityCatalog/FilterBar/FacetDropdown.tsx

```diff
@@ -0,0 +1,196 @@
+  const [anchorWidth, setAnchorWidth] = React.useState<number | undefined>();
+  const menuOpen = Boolean(anchorEl);
+
+  React.useEffect(() => {
+    if (!menuOpen || !anchorEl) return;
+    const update = () => setAnchorWidth(anchorEl.offsetWidth);
+    update();
+    window.addEventListener('resize', update);
+    return () => window.removeEventListener('resize', update);
+  }, [menuOpen, anchorEl]);
+
+  // ----- helpers -----
+  const toggleValue = (value: string) => {
+    const next = new Set(sel);
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    next.has(value) ? next.delete(value) : next.add(value);
+    onChange(facetKey, next);
+  };
+
+  const handleInlineClose = () => {
+    setOpenInline(false);
+    onOpenChange?.(false);
+  };
+
+  const handleMenuClose = () => {
+    setAnchorEl(null);
+    onOpenChange?.(false);
+  };
+
+  // render
+  if (variant === 'inline') {
+    return (
+      <Box>
+        <Button
+          onClick={() => {
+            const next = !openInline;
+            setOpenInline(next);
+            onOpenChange?.(next);
+          }}
+          endIcon={
+            <ExpandMoreIcon
+              sx={{
+                transition: 'transform 0.15s ease',
+                transform: openInline ? 'rotate(180deg)' : 'rotate(0deg)',
+              }}
+            />
+          }
+          variant={isActive ? 'contained' : 'outlined'}
+          color={isActive ? 'primary' : 'inherit'}
+          className={clsx(
+            styles.btnBase,
+            isActive ? styles.btnActiveInline : styles.btnInactiveInline
+          )}
+        >
+          {label}
+        </Button>
+
+        <Collapse in={openInline} unmountOnExit>
+          <ClickAwayListener onClickAway={handleInlineClose}>
+            <Box
+              onMouseLeave={handleInlineClose}
+              className={styles.inlinePanel}
+            >
+              <Box className={styles.gridList}>
+                {options.map(opt => {
+                  const selected = sel.has(opt);
+                  return (
+                    <Chip
+                      key={opt}
+                      label={opt}
+                      onClick={() => toggleValue(opt)}
+                      onDelete={selected ? () => toggleValue(opt) : undefined}
+                      deleteIcon={
+                        selected ? <CloseRoundedIcon fontSize="small" /> : undefined
+                      }
+                      variant={selected ? 'filled' : 'outlined'}
+                      color={selected ? 'primary' : 'default'}
+                      className={clsx(
+                        styles.chip,
+                        selected ? styles.chipSelected : styles.chipUnselected
+                      )}
+                    />
+                  );
+                })}
+              </Box>
+            </Box>
+          </ClickAwayListener>
+        </Collapse>
+      </Box>
+    );
+  }
+
+  // Desktop / Tablet (Menu)
```

Comment:
Ideally, we'll have one implementation that uses responsive design. Can we swap to having one implementation that works on all views?